Lib/test/test_ast/test_ast.py

```diff
@@ -1966,6 +1966,40 @@ def test_ifexp(self):
         for args in (s, l, l), (l, s, l), (l, l, s):
             self.expr(ast.IfExp(*args), "must have Load context")
 
+    def test_ifexp_orelse_stmt(self):
```

Comment:
You could refactor the test by using something like:

```py
msg = "statement given where 'orelse' expression required"
for stmt in [
    "...",
    ...
]:
    self.subTest(stmt), self.assertRaisesRegex(SytanxError, msg):
        ast.parse(stmt)
```

Ditto for the other tests.